main.py

```diff
@@ -1,6 +1,73 @@
-from flask import Flask
+import webbrowser
+from flask import Flask, redirect, url_for, session, request, jsonify
+
+from flask_oauthlib.client import OAuth
+
+
 app = Flask(__name__)
+app.config['GOOGLE_ID'] = "403847176041-1a3qahjkf5m786p54qhu5u5c2qvt0rkt.apps.googleusercontent.com"
+app.config['GOOGLE_SECRET'] = "2YyK88tfPFMDXwtm-2p8kQC9"
```

Comment:
Never hard code ids and secrets. Anyone who has access to the code can see these keys. Since this is open-source, you gave the world access to your Secret key from Google.

Reply:
Ohk will change this ..